src/index.css

```diff
@@ -11,3 +11,145 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* ----- Main styling ----- */
```

Comment:
Even if the CSS is together in only one file, it's easy to understand by how you named the sectioning with comments. I like that.